wpilibj/src/shared/java/edu/wpi/first/wpilibj/PIDController.java

```diff
@@ -115,7 +115,7 @@ public boolean onTarget() {
 
     private PIDController m_controller;
 
-    public PIDTask(PIDController controller) {
+    PIDTask(PIDController controller) {
```

Comment:
https://stackoverflow.com/questions/243218/should-we-declare-a-public-constructor-when-the-class-is-declared-as-package-pri